src/generate_block_metadata.py

```diff
@@ -19,7 +19,7 @@
 
 # Some collection blocks share names with core blocks. We exclude them
 # from the registry for now to avoid confusion.
-BLOCKS_BLACKLIST = {"k8s-job"}
+BLOCKS_BLACKLIST = {"k8s-job", "custom-webhook"}
```

Comment:
```suggestion
BLOCKS_BLOCKLIST = {"k8s-job", "custom-webhook"}
```
I can create an issue for this if you'd prefer to just move forward with your PR.